### ISSUE TITLE
Custom profiles are not loaded in Firefox

### ISSUE THREAD

**Issue by sjieg** [watir/watir#625]
**Steps to reproduce**
1) In IRB execute the following code
```
require 'watir'
require 'selenium-webdriver'
profile = Selenium::WebDriver::Firefox::Profile.new
profile['whatever-setting'] = false
browser = Watir::Browser.new :firefox, :profile => profile
```
2) In the opened browser, navigate to `about:config`
3) Search for "whatever-setting"

**actual**
In Watir 6.6.*
"whatever-setting" can't be found

In Watir 6.4.0:
"whatever-setting" is present and set to "false"

**expected**

"whatever-setting" to be present in Watir 6.6.*

**tested in**

- Selenium 3.4.4 & Selenium 3.5.1
- Firefox 55

**notes**

- You can actually use the text "whatever-setting", Firefox just accepts the setting, even if it's unknown.
- The same issue happens when using `profile = Selenium::WebDriver::Firefox::Profile.new './FF_CUSTOM_PROFILE/'`

**Comment by lmtierney** [comment]
Appears to be a combination of this: 
https://github.com/watir/watir/blob/master/lib/watir/capabilities.rb#L78

And no longer accepting a straight up `profile` key? Seems to expecting the profile to be in an `options` key as well

**Comment by titusfortner** [comment]
Fixed and released in 6.7.1

Supports both `Browser.new :firefox, profile: profile` and `Browser.new :firefox, options: {profile: profile}` but it will throw a deprecation warning about the former because a profile is part of browser options.

**Comment by sjieg** [comment]
Good stuff, thx @titusfortner . If using `options: {profile: profile}` is now best practise, then http://watir.com/guides/firefox/ should be updated.
